docs/operating-scylla/admin.rst

```diff
@@ -93,6 +95,53 @@ The :code:`scylla-server` file contains configuration related to starting up the
 
 .. include:: /operating-scylla/scylla-yaml.inc
 
+.. _object-storage-configuration:
+
+Configuring Object Storage
```

Comment:
Since this uses the same level of header as scylla.yaml above (TOC linked header), the whole block should be moved to just above where `Schedule fstrim` starts. 
So stuff like compression and networking fall under the `scylla.yaml` header
![old](https://github.com/user-attachments/assets/b1877a22-22fa-439f-bb2e-6831a7fa07f0)
and not under the `Configuring Object Storage` header
![new](https://github.com/user-attachments/assets/575af854-f75c-45aa-b7fa-7f6e9061766e)